### PR TITLE
Added functionality to send mail to Supervisor on slot booking

### PR DESCRIPTION
**Pull request by Nivedit24**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from passlib.context import CryptContext
 from functools import wraps
 from wtforms import form
+from flask_mail import Mail, Message
```
Comment (Manvi07) [inline app.py]: Add flask_mail to requirements.txt

File: app.py [hunk 2]
```diff
@@ -389,17 +453,28 @@ def book_slot(EquipID):
...
+        cur_Equi = cur.fetchone()
+        EquipName = cur_Equi['Name']
         mysql.connection.commit()
         cur.close()
         flash("Booking Request Sent.", "success")
+        msg = Message('C4DFED Slot Booking',sender='*********@gmail.com', recipients=[username])
+        msg.body = "Hi {}\nYour booking for the following equipment at C4DFED -\nEquipment: {}\nFrom: {}To: {}\n has been recorded and is under purview by the Admin of C4DFED and your Supervisor: {}.\nYou will be notified of the booking status.".format(name, EquipName, From, To, SupervisorName)
+        mail.send(msg)
+        msg = Message('C4DFED Slot Booking', sender='*********@gmail.com',recipients=[SupervisorEMail])
+        msg.body = "Hi {}\nYour student has booked the following equipment at C4DFED.\nName: {}\nEquipment: {}\nFrom: {}\nTo: {}\nKindly review the booking.".format(SupervisorName,name,EquipName,From,To)
```
Comment (Manvi07) [inline app.py]: `Your student has **sent a booking request.**`

File: app.py [hunk 3]
```diff
@@ -21,6 +22,19 @@
...
+mail = Mail(app)
+
+app.config['MAIL_SERVER'] = 'smtp.gmail.com'
+app.config['MAIL_PORT'] = 465
+app.config['MAIL_USERNAME'] = '********@gmail.com'
+app.config['MAIL_PASSWORD'] = '***********'    
+app.config['MAIL_USE_TLS'] = False
+app.config['MAIL_USE_SSL'] = True
+mail = Mail(app)
+
```
Comment (Manvi07) [inline app.py:38]: To be added in `config.py`.
Can be merged as it is for now since the file is missing in this branch.

File: app.py (hunk 1 above)
Comment (Manvi07) [reply app.py]: Keep both branches to resolve conflict.

**Review by Manvi07 — CHANGES_REQUESTED** [review]

File: app.py [hunk 4]
```diff
@@ -288,6 +349,9 @@ def register():
             cur.execute("INSERT INTO users(name, username, password, accountType) Values(%s, %s, %s, %s)", (name, username, password, accountType))
             mysql.connection.commit()
             flash("Registered Successfully", "success")
+            msg = Message('C4DFED Account', sender='******@gmail.com', recipients=[username])
```
Comment (rishi-s8) [inline app.py]: Why is the sender hardcoded here? It was there at the top config right? We should just use one place where it is configured.

**Review by rishi-s8 — CHANGES_REQUESTED** [review]
@Nivedit24 can we also have the mail being sent at the Acceptance/Rejection by the user after rescheduling? I also mentioned this in the group.

Left some comments. Everything else looks good.

File: app.py (hunk 4 above)
Comment (rishi-s8) [reply app.py]: Same for every other place.

**Review by rishi-s8 — APPROVED** [review]

**Pull request merged**